src/main/java/com/uid2/shared/encryption/AesCbc.java

```diff
@@ -35,6 +36,28 @@ public static EncryptedPayload encrypt(String s, EncryptionKey key) {
         }
     }
 
+    public static EncryptedPayload encrypt(byte[] b, KeysetKey key) {
+        try {
+            final SecretKey k = new SecretKeySpec(key.getKeyBytes(), "AES");
```

Comment:
is this code copied from somewhere else? Can we make reusable functions instead?

Reply:
test